index.js

```diff
@@ -149,11 +151,13 @@ var appServer = function(config) {
 
     self.start = function() {
         // Instantiate up the server
+        //TODO: add i18n support (i18n-node might be a good look)
```

Comment:
I initially put the TODO's so that if anyone else is trying to fix any current issues, they know where in the code to look for. I like the idea of adding the links to the issues, so thanks!